cfgov/data_research/mortgage_utilities/s3_utils.py

```diff
@@ -18,7 +18,7 @@
 S3_SECRET = os.getenv('AWS_S3_SECRET_ACCESS_KEY')
 BASE_BUCKET = settings.AWS_STORAGE_BUCKET_NAME
 MORTGAGE_SUB_BUCKET = "data/mortgage-performance"
-PUBLIC_ACCESS_BASE = 'http://{}.s3.amazonaws.com/{}'.format(
+PUBLIC_ACCESS_BASE = 'https://{}.s3.amazonaws.com/{}'.format(
```

Comment:
See above; the HTTPS format for S3 would be something like

```py
'https://{}/{}'.format(BASE_BUCKET, MORTGAGE_SUB_BUCKET)
```

Reply:
Is as you have written okay, or does it need testing to verify? Fixed in https://github.com/cfpb/cfgov-refresh/pull/3998/commits/6f010183f80d223c28f5d33d179b59cf995d9d86

Reply:
We could add a new test but it'd just be testing that strings are appended correctly. I tested this manually on your branch:

```sh
$ convert_http_to_https* $ ALLOWED_HOSTS='["*"]' DJANGO_SETTINGS_MODULE=cfgov.settings.production cfgov/manage.py shell
>>> from data_research.mortgage_utilities.s3_utils import *
>>> PUBLIC_ACCESS_BASE, S3_SOURCE_BUCKET
(u'https://files.consumerfinance.gov/data/mortgage-performance', u'https://files.consumerfinance.gov/data/mortgage-performance/source')
```

This looks correct to me.